pyramid_oereb/lib/records/geometry.py

```diff
@@ -14,7 +14,7 @@ class GeometryRecord(object):
     Args:
         law_status (pyramid_oereb.lib.records.law_status.LawStatusRecord): The law status of this record.
         published_from (datetime.date): Date from/since when the PLR record is published.
-        geom (Point or LineString or Polygon or MultiPolygon):
+        geom (Point or LineString or Polygon):
```

Comment:
Isn't that confusing to remove MultiLineString and MultiPolygon here when adding MultPolygon support ? 😀

Reply:
Indeed it is. But you know. We are related to the federal data model...

Reply:
Fixed it :stuck_out_tongue_winking_eye: